client/components/AllProducts.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import {SingleProductCard} from './SingleProductCard'
+import {connect} from 'react-redux'
+import {getProductsThunk} from '../store/product'
+
+class AllProducts extends React.Component {
+  componentDidMount() {
+    this.props.getProducts()
+  }
+
+  render() {
+    const {products} = this.props
+    return (
+      <div>
+        {products.map(product => {
+          return (
+            <SingleProductCard
+              key={product.id}
+              name={product.name}
+              price={product.price}
+              imageUrl={product.imageUrl}
+              id={product.id}
```

Comment:
Couldn't this be simplified by passing the product object as props?  In the SingleProductCard you are already deconstructing the props so it would be a case of deconstructing the props.product instead.